subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/DefaultJUnitXmlReport.java

```diff
@@ -46,4 +48,24 @@ public void setOutputPerTestCase(boolean outputPerTestCase) {
     public Property<Boolean> getMergeReruns() {
         return mergeReruns;
     }
+
+    @Override
+    public void setOmitSystemOutLog(boolean omitSystemOutLog) {
```

Comment:
❗ These new flags should be implemented as `Property`s to enable [lazy configuration](https://docs.gradle.org/current/userguide/lazy_configuration.html#lazy_properties).  See the `getMergeReruns()` property avoid for an example to follow.

Reply:
Agree, I'll implement it